projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,24 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			{ /* <table className="screen-reader-text"> */ }
```

Comment:
This can be removed now?

Reply:
Ohh that shouldn't have gotten through. It needs to be there (just not commented out) - nice catch. Fixing up now

Reply:
This is why the table is visible below. Should be better now! (both the <table> with className as well as visible accessibility table gone)